.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+const baseConfig = require('@juntossomosmais/linters/eslint.config.js')
+
+module.exports = {
+  ...baseConfig,
+  root: true,
+  settings: {
+    ...baseConfig.settings,
+    'import/resolver': {
+      typescript: {
+        project: './tsconfig.base.json',
+      },
+    },
+  },
+  overrides: [
+    ...baseConfig.overrides,
+    {
+      files: ['**/*.spec.{js,ts,tsx}'],
+      rules: {
+        'sonarjs/no-duplicate-string': 'off',
```

Comment:
question: make sense add a rule to prevent dependence circle?

like: https://github.com/juntossomosmais/al-khor/blob/master/.eslintrc.js#L15C4-L15C30

Reply:
@JulioCVaz Is a good idea add it in our [shared rules](https://github.com/juntossomosmais/time-out-market/blob/main/packages/linters/src/eslint.config.js)? Maybe we could open a PR ❤️

Reply:
**explaining**: these overrides is just to ignore some rules of `plugin/sonarjs` inside some files